rclcpp_components/src/node_main.cpp.in

```diff
@@ -40,7 +40,7 @@ int main(int argc, char * argv[])
   auto loader = new class_loader::ClassLoader(library_name);
   auto classes = loader->getAvailableClasses<rclcpp_components::NodeFactory>();
   for (const auto & clazz : classes) {
-    std::string name = clazz.c_str();
+    std::string name = clazz;
     if (name.compare(class_name) == 0) {
```

Comment:
Do we actually need this `name` local variable under for loop statement? can we just do the following instead?

```suggestion
    if (clazz.compare(class_name) == 0) {
```

Reply:
I think it is much more better :)